graylog2-server/src/main/java/org/graylog2/inputs/transports/AmqpConsumer.java

```diff
@@ -176,12 +199,35 @@ public void handleDelivery(String consumerTag, Envelope envelope, AMQP.BasicProp
     }
 
     public void connect() throws IOException, TimeoutException {
```

Comment:
could this be private? it changes internal state and afaics won't ever make sense to call directly from another class

Reply:
Good point. Done.